daffodil-lib/src/main/scala/passera/numerics/package.scala

```diff
@@ -30,16 +30,16 @@ import scala.language.implicitConversions
 
 package object numerics {
   implicit def toRicherInt(x: Int): RicherInt = new RicherInt(x)
-  implicit def toRicherInt(x: scala.runtime.RichInt) = new YetRicherInt(
+  implicit def toRicherInt(x: scala.runtime.RichInt): YetRicherInt = new YetRicherInt(
```

Comment:
I would suggest don't bother putting any comments into the passera library code about test coverage that is missing.